packages/console/src/containers/AppContent/components/Document/index.tsx

```diff
@@ -0,0 +1,21 @@
+import useDocumentationUrl from '@/hooks/use-documentation-url';
+
+import DocumentIcon from './DocumentIcon';
+import * as styles from './index.module.scss';
+
+function Document() {
+  const { documentationSiteUrl } = useDocumentationUrl();
+  return (
+    <button
```

Comment:
should be a href link?

Reply:
Maybe we can use the <Button /> component since it supports adding an icon.

Reply:
We do not have button-like link components for now, will add one and replace them. See LOG-5866.